apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/OpenApiDocs.java

```diff
@@ -0,0 +1,191 @@
+package gov.cms.bfd.server.war;
+
+import static gov.cms.bfd.DatabaseTestUtils.HSQL_SERVER_PASSWORD;
+import static gov.cms.bfd.DatabaseTestUtils.HSQL_SERVER_USERNAME;
+import static io.restassured.RestAssured.certificate;
+import static io.restassured.RestAssured.given;
+
+import gov.cms.bfd.DatabaseTestUtils;
+import io.restassured.authentication.AuthenticationScheme;
+import io.restassured.authentication.CertificateAuthSettings;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Set;
+import org.hsqldb.jdbc.JDBCDataSource;
+
+/** Program to gather the OpenAPI yaml content for V1 and V2 and store. */
+public class OpenApiDocs {
```

Comment:
It's using RestAssured, ServerExecutor and the DatabaseTestUtils, but I can move it outside of the test packages and do it without those if that's better.  Leveraging the test framework was suggested in the story so that's what I started with.